.github/workflows/lint.yml

```diff
@@ -35,3 +35,17 @@ jobs:
         uses: pre-commit/action@v3.0.0
         with:
           extra_args: --all-files --hook-stage manual codespell || true
+
+  pep-lint:
+    name: Run pep-lint
```

Comment:
The runtime in CI is reported as "0 s", which means it ran in under a second or two—that seems rather short considering everything its doing. It's great if its actually that fast, but are you _sure_ it's actually checking all the PEPs? I.e., if you introduce a few different issues in various PEP headers, does it report them?

Reply:
It takes ~0.6 seconds on my computer, though we should check that it is running properly on CI. Leaving this unresolved as a marker.

Reply:
Around ~0.22s on this MBP:

```console
❯ time python pep-lint.py
python3 pep-lint.py  0.22s user 0.03s system 97% cpu 0.260 total
```

Reply:
Around ~5s for the full job on CI, with errors: https://github.com/python/peps/actions/runs/5955573732/job/16154498289?pr=3275